cmd/server/http/release.go

```diff
@@ -29,70 +30,50 @@ func release(payload *payload, flowSvc *flow.Service) http.HandlerFunc {
 		logger = logger.WithFields(
 			"service", req.Service,
 			"req", req)
```

Comment:
Would be nice to record the intents here :D

Reply:
Indeed! 👍  Added